libs/langchain-google-cloud-sql-pg/src/DocumentLoader.ts

```diff
@@ -0,0 +1,44 @@
+import PostgresEngine from './engine.js'; 
+import { Document } from "@langchain/core/documents"; 
+import { BaseDocumentLoader } from '@langchain/core/document_loaders/base';
+
+// Options for PostgresLoader
+export interface PostgresLoaderOptions {
+  tableName?: string;
+  schemaName?: string;
+  contentColumns?: string[];
+  metadataColumns?: string[];
+  format?: "text" | "json" | "yaml" | "csv";
+  query?: string;
```

Comment:
We also need metadataJsonColumn. If this column is found it will unnest the metadata into the document. For example if we have metadata columns: age, name with a json column with {address, phone} the resulting metadata will be {age, name, address, phone} rather than {age, name,{address, phone}}